src/Facebook/autoload.php

```diff
@@ -32,6 +32,10 @@
     throw new Exception('The Facebook SDK requires PHP version 5.4 or higher.');
 }
 
+if (defined('FACEBOOK_SDK_POLYFILLS_LOADED') === false) {
```

Comment:
I dont think the constant use useful, you can just require once the polyfills file :)